admin_app/src/components/ContentCard.tsx

```diff
@@ -2,10 +2,29 @@
 import React, { useState } from "react";
 import { PencilIcon, TrashIcon } from "@heroicons/react/20/solid";
 
-export type Content = {
-  content_id: string;
+export enum Language {
+  ENGLISH = "ENGLISH",
+  HINDI = "HINDI",
+}
```

Comment:
Created language Enum

Future: Would be nice to have AAQ-wide config yaml or something from which we construct these enums 🤔